java/src/main/java/org/rocksdb/AbstractWriteBatch.java

```diff
@@ -85,6 +99,19 @@ public void delete(ColumnFamilyHandle columnFamilyHandle, byte[] key)
     delete(nativeHandle_, key, key.length, columnFamilyHandle.nativeHandle_);
   }
 
+  @Override
+  public void delete(ByteBuffer key) throws RocksDBException {
```

Comment:
Can you mark the parameters as `final` please

Reply:
Done